spec/views/devise/sessions/new.html.slim_spec.rb

```diff
@@ -26,7 +26,7 @@
 
     expect(rendered).
       to have_link(
-        t('links.create_account'), href: sign_up_email_path
+        t('links.create_account'), href: sign_up_email_url(request_id: nil)
```

Comment:
We are moving to `_url` for redirects since that is in line with the HTTP spec. AFAIK, you don't need `_url` to include params/